### PR TITLE
BETA: Register proxmox.portkvm.is-a.dev

### PR DESCRIPTION
**Pull request by windows11-on-web**
Added `proxmox.portkvm.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

File: domains/proxmox.portkvm.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "windows11-on-web",
+        "email": "idkalone2019@gmail.com"
+    },
+    "record": {
+        "CNAME": "e4caf830-23f7-44d1-9628-a0cabce7b6dd.cfargotunnel.com"
```
Comment (Stef-00012) [inline domains/proxmox.portkvm.json:7]: I cannot reach your website
"This site can’t be reached"

File: domains/lilycloudrdp-ds.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+    "owner": {
+       "username": "windows11-on-web",
+       "email": "idkalone2019@gmail.com",
+       "discord": "1066707409206267914"
+    },
+
+    "record": {
+        "CNAME": "lilycloudrdp-ds.is-a.dev"
```
Comment (Stef-00012) [inline domains/lilycloudrdp-ds.json:9]: You cannot put the domain itself in the CNAME record

File: domains/systemd-lilyhosting.json [hunk 3]
```diff
@@ -0,0 +1,12 @@
+{
+    "owner": {
+        "username": "windows11-on-web",
+        "email": "idkalone2019@gmail.com"
+    },
+    "record": {
+        "A": ["217.174.245.249"],
```
Comment (Stef-00012) [inline domains/systemd-lilyhosting.json:7]: Hosting by Is-a.dev was shutdown 1 month ago

**Review by Stef-00012 — CHANGES_REQUESTED** [review]

File: domains/proxmox.portkvm.json (hunk 1 above)
Comment (windows11-on-web) [reply domains/proxmox.portkvm.json:7]: My proxmox was offine bro.

Comment (windows11-on-web) [reply domains/proxmox.portkvm.json:7]: @Stef-00012

**Pull request closed without merging**